x-pack/docs/en/security/auditing/ignore-policy.asciidoc

```diff
@@ -0,0 +1,79 @@
+[role="xpack"]
+[[audit-log-ignore-policy]]
+=== Logfile audit events ignore policies
+
+The comprehensive audit trail is necessary to ensure accountability. It offers tremendous
+value during incident response and can even be required for demonstrating compliance.
+
+The drawback of an audited system is represented by the inevitable performance penalty incurred.
+In all truth, the audit trail spends _I/O ops_ that are not available anymore for the user's queries.
+Sometimes the verbosity of the audit trail may become a problem that the event type restrictions,
+<<audit-log-settings, defined by `include` and `exclude`>>, will not alleviate.
+
+*Audit events ignore policies* are a finer way to tune the verbosity of the audit trail.
+These policies define rules that match audit events which will be _ignored_ (read as: not printed).
+Rules match on the values of attributes of audit events and complement the <<audit-log-settings, include/exclude>> method.
+Imagine the corpus of audit events and the policies chopping off unwanted events.
+With a sole exception, all audit events are subject to the ignore policies.
+The exception are events of type `security_config_change`, which cannot be filtered out,
+unless excluded altogether.
+
+IMPORTANT: When utilizing audit events ignore policies you are acknowledging potential
+accountability gaps that could render illegitimate actions undetectable.
+Please take time to review these policies whenever your system architecture changes.
+
+A policy is a named set of filter rules. Each filter rule applies to a single event attribute,
+one of the `users`, `realms`, `roles` or `indices` attributes. The filter rule defines
+a list of <<regexp-syntax,Lucene regexp>>, *any* of which has to match the value of the audit
+event attribute for the rule to match.
+A policy matches an event if *all* the rules comprising it match the event.
+An audit event is ignored, therefore not printed, if it matches *any* policy. All other
+non-matching events are printed as usual.
+
+All policies are defined under the `xpack.security.audit.logfile.events.ignore_filters`
+settings namespace. For example, the following policy named _example1_ matches
+events from the _kibana_system_ or _admin_user_ principals **and** operating over indices of the
```

Comment:
Neither, it's the principals that operate. The phrasing was forced because I tried to fit in the "and" keyword. I've rewritten it, hope it makes more sense like this: 
```
-events from the _kibana_system_ or _admin_user_ principals **and** operating over indices of the
+events from the _kibana_system_ or _admin_user_ principals that operate over indices of the
```